docs/user-guide/features.rst

```diff
@@ -33,6 +33,27 @@ Administrators can also :ref:`upload <create-errata-recipe>` their own errata to
 a repository. Please see the :doc:`recipes` documentation to learn how to
 perform these operations.
 
+Boolean dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+Pulp supports RPM content with `boolean dependencies
+<http://rpm.org/user_doc/boolean_dependencies.html>`_ in the ``Requires`` field
+in these basic repository mirroring use cases:
+
+* synchronization, publication and content upload
+
+* copying content between repositories
+
+* displaying boolean dependencies in the ``Requires`` field
+
+* providing the content to the ``dnf`` consumer to process boolean dependencies
+
+.. Note::
+
+  Pulp itself doesn't process boolean dependencies; a recursive copy won't
```

Comment:
maybe mention that as for now pulp does not depsolve rich deps, but this effort is planned?

Reply:
yeah, missed the planning part ;)